### PR TITLE
Disable arrow key navigation in the drive while modal is active

### PR DESCRIPTION
**Pull request by dariiing**
This PR disables the use of arrow keys inside the drive while a modal is open, fixing #1660.
Previously, arrow keys could still be used to navigate files in the background, which could lead to unintended behavior or conflicts when interacting with the modal. Now, the arrow keys are disabled inside the drive while a modal is opened.

### CODE REVIEW

**Review by Chouhartem — APPROVED** [review]

File: www/common/common-ui-elements.js [hunk 1]
```diff
@@ -2510,13 +2510,11 @@ define([
...
-                return;
-            }
-            if (e.which === 13) {
+              }
+            else if (e.which === 13) {
                 if ($container.find('.cp-icons-element-selected').length === 1) {
                     $container.find('.cp-icons-element-selected').click();
                 }
-                return;
             }
```
Comment (yflory) [inline www/common/common-ui-elements.js:2518]: Try to avoid this type of changes that don't have any impact on the results but only on the code style. I recommend to always run a `git diff` before creating a commit in order to clean unnecessary changes :)

File: www/common/drive-ui.js [hunk 2]
```diff
@@ -1060,6 +1063,7 @@ define([
                             $elements.index($selection.last()[0]);
             var length = $elements.length;
             if (length === 0) { return; }
+
```
Comment (yflory) [inline www/common/drive-ui.js:1066]: Same here, please remove this type of changes from the commit/PR

**Review by yflory — CHANGES_REQUESTED** [review]

**Review by yflory — APPROVED** [review]
